src/utils/index.js

```diff
@@ -28,6 +31,18 @@ const isValidDate = (dateStr) => {
   return moment(dateStr) && date.isValid();
 };
 
+const getCourseUrlSlugPattern = (updatedSlugFlag, courseRunStatuses) => {
+  /**
+   * This function returns the course url slug pattern based on the slugFlag and courseRunStatuses
+   */
+  if (updatedSlugFlag && courseRunStatuses.some((status) =>
+    // eslint-disable-next-line implicit-arrow-linebreak
+    (IN_REVIEW_STATUS.includes(status) || POST_REVIEW_STATUSES.includes(status)))) {
```

Comment:
During the review, the course/course run is not editable. The validation during that phase will not let course run's review move ahead.

Reply:
We will be autogenerating the course `url_slug` on submitting the course-run for review (from the backend). The `url_slug` will already be in a valid form.